src/core/field.js

```diff
@@ -84,7 +84,11 @@ export default (options: FieldOptions = {}) => (Input: Component): Component =>
       } else if ('defaultValue' in props) {
         // something was changed or an initial call
         if (isInitialCall) {
-          this.stateManager.setValue(this.value || props.defaultValue, triggerOnChange);
+          // if field has an initial value set (even falsy ones) use it. otherwise default
+          this.stateManager.setValue(
+            this.value !== undefined ? this.value : props.defaultValue,
```

Comment:
what if the value is `null`?

Reply:
`null` seems like intended behavior. I didn't want to touch it, in case the user does, in fact, want to purposefully convey a lack of value, other than just boolean `false`